website/content/docs/install/index.mdx

```diff
@@ -163,250 +107,158 @@ $ sudo yum -y install nomad
-</CodeBlockConfig>
+<Tab heading="Windows" group="windows">
 
-<h3>Verify cgroup controllers</h3>
+<Tabs>
+<Tab heading="Chocolatey" group="windows-chocolatey">
 
-On Linux, Nomad uses cgroups to control resource usage of tasks. If one or more
-required cgroups are unavailable, Nomad will disable resource controls that
-require cgroups entirely. With cgroups v2, you can verify that you have all
-required controllers as follows:
+[Chocolatey](https://chocolatey.org/) is a free and open-source package
+management system for Windows. Install the [Nomad
+package](https://chocolatey.org/packages/nomad) from the command-line.
 
 ```shell-session
-$ cat /sys/fs/cgroup/cgroup.controllers
-cpuset cpu io memory pids
+$ choco install nomad
 ```
 
-See the documentation on [cgroup controller requirements][] for more details.
-
-</Tab>
-
-<Tab heading="macOS">
-<h3>Add the Nomad binary to your system path</h3>
+-> Chocolatey and the Nomad package are **NOT** directly maintained
+by HashiCorp. The latest version of Nomad is always available by manual
+installation.
 
-Permanently add a new location to your path by editing your shell's settings
-file (usually called something like `~/.bashrc`, where the part of the filename
-after the `.` and before `rc` is the name of your shell). In that file you
-should see a line that starts with `export PATH=`, followed by a
-colon-separated list of locations. Add the location of the Nomad binary to that
-list and save the file. Then reload your shell's configuration with the command
-`source ~/.bashrc`, replacing `bash` with the name of your shell.
 </Tab>
+<Tab heading="Manual" group="windows-manual">
 
-<Tab heading="Windows">
+@include 'install/manual-install.mdx'
 
-<h3>Add the Nomad binary to your system path</h3>
-
-Add a location to your path via the GUI by navigating to `Environment
-Variables` in your system settings, and looking for the variable called `PATH`.
-You should see a semicolon-separated list of locations. Add the Nomad binary's
-location to that list and then launch a new console window.
 </Tab>
 </Tabs>
-</Tab>
 
-<Tab heading="Homebrew on macOS" group="homebrew">
-No additional steps necessary after installing Nomad using Homebrew.
 </Tab>
 
-<Tab heading="Chocolatey on Windows" group="chocolatey">
-No additional steps necessary after installing Nomad using Chocolatey.
-</Tab>
+</Tabs>
 
-<Tab heading="Linux" group="linux">
 
-Note that if you are running Nomad on Linux, you'll need to run client agents as
-root (or with `sudo`) so that cpuset accounting and network namespaces work
-correctly.
+## Post-installation steps
 
-<h3>Install CNI plugins</h3>
+These steps are optional but can be helpful for running Nomad and to
+take advantage of additional Nomad functionalities.
 
-Nomad uses CNI plugins to configure network namespaces when using the `bridge`
-network mode. All Linux Nomad client nodes using network namespaces must have
-CNI plugins installed.
+<Tabs>
+<Tab heading="Linux" group="linux">
```

Comment:
In the post install section, I removed the separate _Add the Nomad binary to your system path_ content about adding the binary to the path because the manual install directions already contain that verbiage.